integration-cli/docker_cli_logs_test.go

```diff
@@ -178,6 +178,41 @@ func (s *DockerSuite) TestLogsSince(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestLogsUntil(c *check.C) {
```

Comment:
@vdemeester Okay, but why are the current CLI tests still there (and pass)? I looked at the `docker/cli` library too, and couldn't find any integration tests - so should these tests just be deleted rather than moved?